src/main/kotlin/org/malachite/estella/people/api/UserController.kt

```diff
@@ -49,35 +48,35 @@ class UserController(
     fun getLoggedInUser(@CookieValue("jwt") jwt: String?): ResponseEntity<Any> {
         val user = securityService.getUserFromJWT(jwt)
         return user?.let {
-            ResponseEntity.ok(user)
-        } ?: ResponseEntity.status(401).body("Unauthenticated")
+            ResponseEntity(user, HttpStatus.OK)
+        } ?: ResponseEntity(Message("Unauthenticated"), HttpStatus.UNAUTHORIZED)
```

Comment:
Skoro mamy SuccessMessage można by też dać UnauthenticatedMessage

Reply:
Znaczy SuccessMessage dałem tylko dlatego, że ten success był kilka razy - unathenticated jest tylko raz, więc według mnie nie bardzo jest sens.

Reply:
Znaczy ogólnie mi się wydaje że będzie w każdym kontrolerze jak będziemy sprawdzać czy jest token XD

Reply:
Pewnie tak, ale to wtedy się zmieni XD